app/controllers/dorms_controller.rb

```diff
@@ -56,8 +56,7 @@ def show
       @is_student = !@student.nil?
     end
 
-    @period = current_draw_period()
-    puts "PERIOD", @period
+    @period = current_draw_period ? true : false
```

Comment:
Why the `value? true: false`?  Is `current_draw_period` a boolean?  If so, this is unnecessary and should be removed.  If not, it looks like there's a lot of stuff elsewhere we need to change to this.

Reply:
It's because of the line in `dorms/show.html.erb.` which says `var period = <%= @period %>;`.  In most instances non-nil values are falsey, but in HTML files if @period is nil it will throw an error.